[IMP] accounting: update vendor bills tutorial link (#12382)
odoo/documentation: content/applications/finance/accounting/vendor_bills.rst

Vendor bills
============

Vendor bills can be registered either **manually** or **automatically** in Odoo. The
:ref:`Aged Payable report <accounting/vendor_bills/age-payable-report>` provides an overview of all
outstanding bills to help ensure timely payment of the correct amounts.

.. seealso::
   - Tutorial `Registering a vendor bill <https://www.odoo.com/fr_FR/slides/slide/register-a-vendor-bill-6582>`_
   - :doc:`/applications/inventory_and_mrp/purchase/manage_deals/manage`

.. _accounting/vendor_bills/creation:

Bill creation
=============

.. _accounting/vendor_bills/creation-manual:

Manually
--------

To create a vendor bill manually, go to :menuselection:`Accounting --> Vendors --> Bills` and
click :guilabel:`Create`.

> 

Author: Will do that ;)

<invoice-digitization/email-alias>` associated with the purchase journal, or by **uploading a PDF**
in :menuselection:`Accounting --> Vendors --> Bills` and then clicking :guilabel:`Upload`.

.. _accounting/vendor_bills/bill-completion:

Bill completion
===============

Whether the bill is created manually or automatically, make sure the following fields are
appropriately completed:

- :guilabel:`Vendor`: Odoo automatically fills in some information based on the vendor's registered

>   information, previous purchase orders, or bills.

Author: Thanks for the clarification!

- :guilabel:`Bill Reference`: Add the sales order reference provided by the vendor, and is used to
  :ref:`match <accounting/payments/matching>` the products when they are received.
- :guilabel:`Auto-Complete`: Select a past bill/purchase order to complete the document
  automatically. The :guilabel:`Vendor` field should be completed before completing this field.
- :guilabel:`Bill Date`: Select the document's issuance date.
- :guilabel:`Accounting Date`: Update the document's accounting registration date if needed.
- :guilabel:`Payment Reference`: Automatically indicated in the :guilabel:`Memo` field when
   registering the payment.
- :guilabel:`Recipient Bank`: Indicates the account number to make the payment.
- :guilabel:`Due Date` or :guilabel:`Terms` for paying the bill.

> - :guilabel:`Journal`: Select which journal should record the bill and in which :doc:`Currency
>   <get_started/multi_currency>`.

Reviewer: ```suggestion
- :guilabel:`Journal`: Select which journal should record the bill and in which :doc:`currency
  <get_started/multi_currency>`.
```

     Purchase (IAP) </applications/essentials/in_app_purchase>` credit or tokens.

.. _accounting/vendor_bills/bill-completion:

Bill completion
===============

Whether the bill is created manually or automatically, make sure the following fields are
appropriately completed:

- :guilabel:`Vendor`: Odoo automatically fills in some information based on the information on the
  vendor's contact record as well as previous purchase orders and bills.
- :guilabel:`Bill Reference`: Add the sales order reference provided by the vendor. This field is
  used to :ref:`match <accounting/payments/matching>` the products when they are received.
- :guilabel:`Auto-Complete`: Select a past bill/purchase order to complete the document
  automatically. The :guilabel:`Vendor` field should be completed before completing this field.
- :guilabel:`Bill Date`: Select the document's issuance date.
- :guilabel:`Accounting Date`: Update the document's accounting registration date if needed.
- :guilabel:`Payment Reference`: Automatically indicated in the :guilabel:`Memo` field when
   registering the payment.
- :guilabel:`Recipient Bank`: Indicates the account number to which the payment will be made. This

>   field is required when paying via batch payment files (such as NACHA and :doc:`SEPA

Reviewer: That's the plan but only from 17.0 as the localization doc doesn't exist in 16.0 (even though the feature does) :)

>   field is required when paying via batch payment files (such as NACHA and :doc:`SEPA

Reviewer 2: @dade-odoo my bad, sorry!

>   field is required when paying via batch payment files (such as NACHA and :doc:`SEPA

Reviewer: No worries @auva-odoo ! I made the same comment earlier :joy: just trying to save Anne Françoise from having to repeat herself

   changes are required.

.. _accounting/vendor_bills/bill-payment:

Payment and reconciliation
==========================

To register a payment, click on :guilabel:`Register Payment`. In the :guilabel:`Register Payment`
window, select the :guilabel:`Journal`, the :guilabel:`Payment Method`, the :guilabel:`Amount`, and
the :guilabel:`Currency`.

When the :guilabel:`Amount` paid is less than the total remaining amount on the vendor bill, the
payment is :ref:`partial <accounting/payments/partial-payment>`, and the :guilabel:`Payment
Difference` field displays the outstanding balance.

The :guilabel:`Memo` field is filled automatically if the :guilabel:`Payment Reference` has been set
correctly in the vendor bill. If the field is empty, select the vendor invoice number as a
reference.

Then click :guilabel:`Create payment`. An :guilabel:`In Payment`/:guilabel:`Partial` banner appears
on the bill until it is :doc:`reconciled <bank/reconciliation>`, at which point the status updates
to :guilabel:`Paid`.

.. seealso::
   - :doc:`payments`
   - :doc:`bank/reconciliation`

.. _accounting/vendor_bills/age-payable-report:

Aged payable report
===================

> To get an overview of the open vendor bills and related due dates, go to :menuselection:`Accounting

Reviewer 2: ```suggestion
To get an overview of the open vendor bills and their due dates, go to :menuselection:`Accounting
```
nitpick

--> Reporting --> Aged payable`.

Click the :icon:`fa-caret-right` :guilabel:`(right arrow)` icon next to a vendor to view the details

> of all outstanding bills, including the due dates and amounts.

Reviewer 2: ```suggestion
of all their outstanding bills, including the due dates and amounts.
```
since it's only the bills from this particular vendor, not all bills

.. note::
   - Bills can be :doc:`digitized <vendor_bills/invoice_digitization>` for
     automatic completion by clicking :guilabel:`Send for Digitization`.
   - Once the bill is uploaded, the PDF document appears on the right side of the screen, making it

>      easy to fill in the bill information.

Reviewer: Move this note up so it comes at the end of the "Automatically" section.

Author: You're right, it makes more sense! I also added the info about IAP for document digitization.

.. _accounting/vendor_bills/bill-confirmation:

Bill confirmation
=================

Click :guilabel:`Confirm` when the document is completed. The status changes to :guilabel:`Posted`,

> and a journal entry is generated based on the vendor bill configuration.

Reviewer: ```suggestion
and a journal entry is generated based on the vendor bill information.
```
"configuration" confuses me a bit. I think "information" is better - let me know if you disagree!

Author: Sure! It's just that we love to use "configuration" in French in these cases :)

.. note::

>    Once confirmed, it can no longer be updated. Click :guilabel:`Reset to draft` if

Reviewer: ```suggestion
   Once confirmed, a vendor bill can no longer be updated. Click :guilabel:`Reset to draft` if
```

   changes are required.

.. _accounting/vendor_bills/bill-payment:

Payment and reconciliation
==========================

To register a payment, click on :guilabel:`Register Payment`. In the :guilabel:`Register Payment`
window, select the :guilabel:`Journal`, the :guilabel:`Payment Method`, the :guilabel:`Amount`, and
the :guilabel:`Currency`.

When the :guilabel:`Amount` paid is less than the total remaining amount on the vendor bill, the
payment is :ref:`partial <accounting/payments/partial-payment>`, and the :guilabel:`Payment
Difference` field displays the outstanding balance.

The :guilabel:`Memo` field is filled automatically if the :guilabel:`Payment Reference` has been set
correctly in the vendor bill. If the field is empty, select the vendor invoice number as a
reference.

Then click :guilabel:`Create payment`. An :guilabel:`In Payment`/:guilabel:`Partial` banner appears

> on the bill until it is :doc:`reconciled <bank/reconciliation>`.

Reviewer: ```suggestion
on the bill until it is :doc:`reconciled <bank/reconciliation>`, at which point the status updates to :guilabel:`Paid`.
```

.. seealso::
   - :doc:`payments`
   - :doc:`bank/reconciliation`

.. _accounting/vendor_bills/age-payable-report:

Aged payable report
===================

To get an overview of the open vendor bills and related due dates, go to :menuselection:`Accounting
--> Reporting --> Aged payable`.

Click the :icon:`fa-caret-right` :guilabel:`(right arrow)` icon next to a vendor to view the details
of all outstanding bills, including the due dates and amounts.

.. Note::

>    - To export and save the information as a PDF or XLSX file, click :guilabel:`Save`.

Reviewer: In 16, we have the Save button.
In 17, the Save option is behind the drop-down icon.
In 18, it's replaced entirely by the Copy to Documents option.
In all of them, we have individual buttons for both PDF and XLSX, so let's use those instead :)

   - Multiple bills for the same purchase order may be issued if the vendor is on back-order and
     sends invoices as products are shipped or if the vendor sends partial bills or requests a

>      deposit.

Reviewer: This item seems out of place. There's not a great place for it, but I think it's most related to the match feature. Could this bullet point be a separate note that comes after the screenshot and be reworded to make more sense about why it's placed there? Something like

> Multiple bills for the same purchase order may be issued if the vendor is on back-order and sends invoices as products are shipped or if the vendor sends partial bills or requests a deposit. In this case, multiple bills may have the same :guilabel:`Bill Reference`.
